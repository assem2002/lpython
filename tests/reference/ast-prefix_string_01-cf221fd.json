{
    "basename": "ast-prefix_string_01-cf221fd",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/errors/prefix_string_01.py",
    "infile_hash": "0d83c0e32a78023fccb343a4d3358071792265c1ae357176fe0912eb",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast-prefix_string_01-cf221fd.stderr",
    "stderr_hash": "01778a489f20dcff14eb922861f176cb7b5998337b0688af164af6e2",
    "returncode": 1
}