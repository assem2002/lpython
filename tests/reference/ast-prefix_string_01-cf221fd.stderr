syntax error: Token '"Hello"' (of type 'string') is unexpected here
 --> tests/errors/prefix_string_01.py:3:11
  |
3 |     print "Hello", "World" 
  |           ^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
